include/SSVOpenHexagon/Components/CCustomWall.hpp

```diff
@@ -27,6 +27,8 @@ class CCustomWall
 private:
     std::array<sf::Vector2f, 4> vertexPositions;
     std::array<sf::Color, 4> vertexColors;
+    bool canCollide{true};
+    int8_t renderOrder{1};
```

Comment:
I'm assuming this is not used at the moment -- please comment it out so that it doesn't take additional memory until we figure out how to do this properly.

Reply:
Like I mentioned in the original post, when wallAccs decelerate and start traveling away from the center, they never get deleted and as a result, this is why pack developers couldn't rely on the native method of incrementing to increment their levels. (Trust me, you'll see why I did this when I start messing with acceleradiant).

A timer would be way more difficult to implement. How would we know how long it takes for the wall to travel to the center (even if we account a change in speed)? How do we ensure the wall doesn't get terminated too early? I feel like trying to make all of those calculations will cost more performance than the current implementation. I think the current implementation is better (but I will weed out some calculations because I'm reverting some changes).

Reply:
Oh ffs ofc this replied to the wrong comment. Thanks GitHub mobile!